Automattic/jetpack
# RNA Base styles - Move RNA styles to base-styles package.

Removes repeated `rna-styles.scss` files and imports them from the base-styles package instead.

#### Changes proposed in this Pull Request:
* Adds a `style.scss` in `base-styles` package
* Updates all corresponding imports to use those styles instead
* Removes duplicate `rna-styles.scss`

#### Jetpack product discussion


p9dueE-3MR-p2

It's important to note that **this PR does not solve the issues** mentioned in the Jetpack product discussion. 
It simply handles the multiple repeated `rna-styles` found in many packages/plugins by importing them from a single place.
Further optimizations, including:

1. handling global styles, 
2. breaking down the `base-styles/style.scss` to partials and 
3. importing ONLY what we need in component styles

should be handled in follow up PRs. 

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:
- Ensure nothing has visually changed in the Jetpack-plugin Connect Screen
- Ensure nothing has visually changed in the Backup plugin
- Run storybook and verify all components are also the same

Finally look at the code to ensure we didn't miss any variables defined only in one of the multiple `rna-styles`.
